FreifunkMemorandumofUnderstanding_it.md

```diff
@@ -0,0 +1,67 @@
+
+## Preambolo
+
+Le reti (wireless) libere sono costruite e messe a disposizione da una moltitudine di iniziative locali. Gli utenti sono al contempo operatori di reti di computer. Essi creano reti "fai da te", connettendo con le proprie forze appartamenti, case, vie, quartieri, paesi, o intere città. Una rete Freifunk è costruita in maniera decentralizzata e viene gestita da molti individui. Questa struttura organizzativa decentralizzata favorisce deliberatamente la volontà di attivarsi in loco invece di farsi governare da una entità più in alto. "Frei" (libero) sta a significare che queste reti sono fatte per essere aperte, accessibili anonimamente, gestite in ottica non commerciale, e le informazioni che in essa viaggiano non devono essere ispezionate, modificate o censurate. Sebbene Freifunk si riferisca principalmente a reti WLAN, il termine "freie Netzwerke" (reti libere) dovrebbe essere inteso in senso più ampio. Tra le altre cose, aderisce allo spirito della definizione fornita da freenetworks.org.
+
+Nonostante il livello di decentralizzazione, riteniamo ragionevole e importante arrivare ad un'intesa comune sui principi fondanti dell'iniziativa Freifunk, allo scopo di poter agire in congiuntura e supportarci l'un l'altro nelle nostre attività.
+
+## Obbiettivi di Freifunk
+
+L'uso di reti tecnologiche è diventata una pratica comune già molto tempo fa, ma nonostante ciò non si riflette a sufficienza su come si agisce al loro interno e sulle dinamiche di potere sottostanti. Freifunk ha tra i suoi obbiettivi dichiarati quello di rendere una fetta più ampia della società consapevole di questi aspetti, di promuovere l'accesso libero e aperto a reti pubbliche, e di incoraggiare ad agire in prima persona e con la propria testa. Ci piacerebbe che questo processo fosse inclusivo nei confronti dei bisogni di una moltitudine di gruppi sociali. Riteniamo che la diversità e uno spettro ampio di idee siano fondamenta importanti da cui partire per raggiungere questi obbiettivi. Vogliamo incoraggiare tutti i partecipanti a prendere in mano la situazione, dando loro una mano a prendere confidenza con queste tematiche. Ci rendiamo conto che apprendere tutto questo è complesso e richiede tempo, ma riteniamo importante evitare che il desiderio da parte di alcuni partecipanti di limitarsi a "consumare" il servizio finisca col produrre gerarchie tra chi ha acquisito delle conoscenze e chi no. Per come vediamo le cose, questo porta ad erodere il concetto alla base di una rete "fai da te".
```

Comment:
_ad agire in prima persona e **pensare** con la propria testa._

_Ci piacerebbe che questo processo fosse inclusivo nei confronti dei bisogni di una moltitudine di gruppi sociali._
_**In questo processo vogliamo tener conto dei bisogni di diversi gruppi sociali**_ (almeno dal testo tedesco io capisco questo)